workbench-core/example/infrastructure/integration-tests/support/resources/dynamicAuthorization/group.ts

```diff
@@ -29,6 +31,11 @@ export default class Group extends Resource {
     return this._axiosInstance.get(`${this._api}/is-user-assigned/${userId}`);
   }
 
+  public async delete(): Promise<void> {
+    this._clientSession.removeCleanupTask(`group-${this.id}`);
```

Comment:
if the delete API request isnt successful, we dont want to remove the cleanup task.